packages/date-picker/src/utils/isDisabledDate.ts

```diff
@@ -0,0 +1,15 @@
+import { Modifier } from "react-day-picker/types/Modifiers"
+import { ModifiersUtils } from "react-day-picker"
+import { defaultCalendarClasses } from "../DatePicker/components/Calendar/CalendarClasses"
+
+export const isDisabledDate = (
+  parsedDate: Date,
+  disabledDays: Modifier | Modifier[]
+): boolean => {
+  // Check if typed day has disabled modifier
+  const isTypedDayDisabled = ModifiersUtils.getModifiersForDay(parsedDate, {
+    [defaultCalendarClasses.disabled]: disabledDays,
+  })
+
+  return isTypedDayDisabled.length !== 0
```

Comment:
```suggestion
  const selectedDayDisabledModifiers = ModifiersUtils.getModifiersForDay(parsedDate, {
    [defaultCalendarClasses.disabled]: disabledDays,
  })

  return selectedDayDisabledModifiers.length !== 0
```

Comment is redundant, and the variable name sounded like a boolean despite being an array.